### PR TITLE
List: add bottom margin for last item

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: e2e/testcafe-devextreme/tests/editors/selectBox/common.ts [hunk 1]
```diff
@@ -1,14 +1,67 @@
...
+} from '../../../helpers/domUtils';
 import url from '../../../helpers/getPageUrl';
 import SelectBox from 'devextreme-testcafe-models/selectBox';
 import { createWidget } from '../../../helpers/createWidget';
-import { appendElementTo, setStyleAttribute } from '../../../helpers/domUtils';
 
 fixture.disablePageReloads`SelectBox placeholder`
   .page(url(__dirname, '../../container.html'));
 
+test('Last Item from last group in dropdown list has margin-bottom = 4px in Fluent Theme', async (t) => {
```
Comment (EugeniyKiyashko) [inline e2e/testcafe-devextreme/tests/editors/selectBox/common.ts]: Let's have a test on the list, because this test is no guarantee that it works correctly for other ddEditors.

File: packages/devextreme/scss/widgets/fluent/list/_sizes.scss [hunk 2]
```diff
@@ -13,6 +14,7 @@ $fluent-list-next-button-horizontal-padding: null !default;
 
 @if $size == "default" {
   $fluent-list-vertical-padding: 4px !default;
+  $fluent-list-bottom-padding: 4px !default;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/scss/widgets/fluent/list/_sizes.scss]: We shouldn't concrete values for 'default', 'compact' themes. They are equal. $fluent-list-vertical-padding values too. Let's refactor this part.

File: e2e/testcafe-devextreme/tests/editors/list/grouping.ts [hunk 3]
```diff
@@ -53,6 +54,37 @@ test('Grouped list appearance', async (t) => {
   },
 }));
 
+test('Grouped List last item of last group should have margin-bottom = 4px', async(t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  
+  const items = Selector('.dx-item');
```
Comment (EugeniyKiyashko) [inline e2e/testcafe-devextreme/tests/editors/list/grouping.ts]: Let's use List model for this.

Comment (nikkithelegendarypokemonster) [reply e2e/testcafe-devextreme/tests/editors/list/grouping.ts]: [640d4e0](https://github.com/DevExpress/DevExtreme/pull/27093/commits/640d4e0d7b58662795cac821b7a1415c4d807f6d)

File: packages/devextreme/scss/widgets/fluent/list/_sizes.scss (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply packages/devextreme/scss/widgets/fluent/list/_sizes.scss]: [640d4e0](https://github.com/DevExpress/DevExtreme/pull/27093/commits/640d4e0d7b58662795cac821b7a1415c4d807f6d)

File: e2e/testcafe-devextreme/tests/editors/selectBox/common.ts [hunk 4]
```diff
@@ -1,10 +1,12 @@
 import { Selector } from 'testcafe';
 import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
 import { testScreenshot } from '../../../helpers/themeUtils';
+import {
```
Comment (EugeniyKiyashko) [inline e2e/testcafe-devextreme/tests/editors/selectBox/common.ts]: Please, revert unnecessary changes

File: e2e/testcafe-devextreme/tests/editors/list/grouping.ts [hunk 5]
```diff
@@ -53,6 +54,37 @@ test('Grouped list appearance', async (t) => {
   },
 }));
 
+test('Grouped List last item of last group should have margin-bottom = 4px', async(t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  const list = new List('#container');
+  const lastItem = list.getItems().length - 1;
```
Comment (EugeniyKiyashko) [inline e2e/testcafe-devextreme/tests/editors/list/grouping.ts]:
```suggestion
  const lastItemIndex = list.getItems().length - 1;
```

File: e2e/testcafe-devextreme/tests/editors/list/grouping.ts [hunk 6]
```diff
@@ -53,6 +54,37 @@ test('Grouped list appearance', async (t) => {
...
+  await t.hover(list.getItem(lastItem).element);
+
+  await testScreenshot(t, takeScreenshot, 'Grouped List with correct margin bottom.png', { element: '#container' });
+
+  await t
+  .expect(compareResults.isValid())
+  .ok(compareResults.errorMessages());
+}).before(async () => createWidget('dxList', {
+    dataSource: [{
```
Comment (EugeniyKiyashko) [inline e2e/testcafe-devextreme/tests/editors/list/grouping.ts]: Let's make this config shorter and leave only two groups

File: e2e/testcafe-devextreme/tests/editors/list/grouping.ts [hunk 7]
```diff
@@ -53,6 +54,37 @@ test('Grouped list appearance', async (t) => {
   },
 }));
 
+test('Grouped List last item of last group should have margin-bottom = 4px', async(t) => {
```
Comment (EugeniyKiyashko) [inline e2e/testcafe-devextreme/tests/editors/list/grouping.ts]: margin-bottom = 4px it's actual value for Fluent theme only. I think that name can be more common with description what appearance testing.

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]
